src/main/java/com/google/songgame/servlets/UserServlet.java

```diff
@@ -49,9 +53,61 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     datastore.put(userEntity);
   }
 
+  @Override
+  public void doPut(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Map<String, String> roomProperties = readJSONFromRequest(request);
+
+    Query userQuery =
+        new Query("User").addFilter("userId", FilterOperator.EQUAL, roomProperties.get("userId"));
+    PreparedQuery result = datastore.prepare(userQuery);
+    Entity currentUser = result.asSingleEntity();
+
+    // Add current roomId to current user.
+    currentUser.setProperty("roomId", roomProperties.get("roomId"));
```

Comment:
We have a list of users in the room, so is it necessary to include the roomId in the Users property? What benefits does it have over a list of the users per room, and what costs does it have compared to the list.